gradle/libs.versions.toml

```
@@ -149,6 +150,7 @@ boms-micronaut-kubernetes = { module = "io.micronaut.kubernetes:micronaut-kubern
 boms-micronaut-langchain4j = { module = "io.micronaut.langchain4j:micronaut-langchain4j-bom", version.ref = "managed-micronaut-langchain4j" }
 boms-micronaut-liquibase = { module = "io.micronaut.liquibase:micronaut-liquibase-bom", version.ref = "managed-micronaut-liquibase" }
 boms-micronaut-logging = { module = "io.micronaut.logging:micronaut-logging-bom", version.ref = "managed-micronaut-logging" }
+boms-micronaut-mcp = { module = "io.micronaut.mcp:micronaut-mcp-server-java-sdk-bom", version.ref = "managed-micronaut-mcp" }
```

You are using the wrong dependency: 
https://central.sonatype.com/artifact/io.micronaut.mcp/micronaut-mcp-bom
```suggestion
boms-micronaut-mcp = { module = "io.micronaut.mcp:micronaut-mcp-bom", version.ref = "managed-micronaut-mcp" }
```